packages/vscode-extension-pack-kogito-kie-editors/it-tests/helpers/dmn/ExpressionEditorHelper.ts

```diff
@@ -42,7 +42,9 @@ export default class ExpressionEditorHelper {
     await assertWebElementIsDisplayedEnabled(title);
 
     const type = await boxedExpressionEditor.findElement(
-      By.xpath(`//div[@data-ouia-component-type='expression-column-header-cell-info']/p[contains(@class, 'data-type')]`)
+      By.xpath(
+        `//div[@data-ouia-component-type='expression-column-header-cell-info']/p[text()='${expectedTitle}']/following-sibling::p`
```

Comment:
The idea is to get the sibling <p> element of the title element, which is the data-type.